appserver/payara-appserver-modules/security-openid/src/main/java/fish/payara/security/openid/OpenIdUtil.java

```diff
@@ -66,7 +66,9 @@ private OpenIdUtil() {
         if (configResult.isPresent()) {
             return configResult.get();
         }
-        result = (T) TranslatedConfigView.getTranslatedValue(result);
+        if (type == String.class) {
+            result = (T) TranslatedConfigView.expandValue((String) result);
+        }
```

Comment:
Is the _else_ case handled elsewhere?

Reply:
`getTranslatedValue` does transformation only on Strings. I think the story is, that it inherited too general signature, when it was extracted from the interceptor-style method `invoke` before 2011.